subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include "hal/radio_df.h"
+#include "lll_df.h"
+
+/* @brief Function performs common steps for initialization and reset
```

Comment:
These doxygen-style comments are using `/*` instead of `/**`. Is that intentional?

Reply:
This is intentional, because there is no doxygen documentation generated for controller internal code. I wanted to add information about the function but I didn't want to generate documentation for that. 
Nevertheless I went to use doxygen markups to:
- have similar style of comments as in other header files
- if we decide to provide documentation for internals then simple change is required.

Reply:
It doesn't hurt to make these `/**`, though I find that confusing for internal API as it normally suggests to me it's meant to be public.

I too use doxygen markup (e.g. `<at>param`) for internal functions in standard comment blocks, but I don't use `<at>brief` because it's annoying (and unnecessary with [JAVADOC_AUTOBRIEF](https://www.doxygen.nl/manual/config.html#cfg_javadoc_autobrief) which Zephyr didn't enable), so maybe people haven't noticed....

Reply:
I think that we have to decide about common policy for internal API documentation.
In my humble opinion, it is good to have it, especially in cases where functions does more than name of the function suggest.

I leave it as is now. If it is requested to change, I'll do it throughout all my code in separate PR.

Should I create a github issue about internal API documentation format?